tools/check_format.py

```diff
@@ -141,6 +141,19 @@ def checkNamespace(file_path):
   return []
 
 
+def checkJavaProtoOptions(file_path):
+  java_multiple_files = False
+  java_package_correct = False
+  for line in fileinput.FileInput(file_path):
+    if "option java_multiple_files = true;" in line:
+      java_multiple_files = True
+    if "option java_package = \"io.envoyproxy.envoy" in line:
+      java_package_correct = True
+    if java_multiple_files and java_package_correct:
+      return []
+  return ["Java proto options are not set correctly for file: %s" % file_path]
```

Comment:
Can we output what options are missing? Would save people from having to grep around to figure out what's wrong

Reply:
Added more detailed error messages.